stdpopsim/ext/selection.py

```diff
@@ -41,7 +41,7 @@ def __attrs_post_init__(self):
                 )
         elif self.distribution_type == "e":
             # An exponentially-distributed fitness effect (mean).
-            # A negative value for the mean is permitted.
+            # A negative value for the mean is not permitted.
```

Comment:
ok - but then how do we do an exponential distribution of deleterious things?

Reply:
Good catch. Yes, `rexp()` does accept negative values for the mean, and we do need this.